### ISSUE TITLE
Opt-in region support and regional STS endpoints

### ISSUE THREAD

**Issue by amlodzianowski** [okta/okta-aws-cli#117]
Hello,

Thank you for the hard work on this project! This CLI has been super useful.

I ran into an issue today when trying to access one of the AWS opt-in regions using the CLI today. It seems to me that that the legacy/global STS endpoint is being used to retrieve credentials, which does not work for opt-in regions by default:
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
https://docs.aws.amazon.com/sdkref/latest/guide/feature-sts-regionalized-endpoints.html
The default IAM setting in each AWS account for the global STS endpoint is "Valid only in AWS Regions enabled by default". 

Toggling this setting on an account allows okta-aws-cli to work with opt-in regions, but it would be great to be able to have the ability to specify whether the global or regional STS endpoint should be used when retrieving credentials. Regional endpoints are also the recommended approach by AWS.

I tried setting the `AWS_STS_REGIONAL_ENDPOINTS=regional` env var before invoking okta-aws-cli but it seemed to have no effect.

Steps to reproduce:
- Retrieve env var credentials using okta-aws-cli
- Run `export AWS_DEFAULT_REGION=af-south-1` (or any other opt-in region)
- Run `aws ec2 describe-instances`. The following error should be returned:
```
An error occurred (AuthFailure) when calling the DescribeInstances operation: AWS was not able to validate the provided access credentials
```

okta-aws-cli version 1.1.0

Please let me know if there is any other information I can provide here.

**Comment by monde** [comment]
This is important info, thanks for detailing it out @amlodzianowski . I'm working on a headless feature for okta-aws-cli and will be sure to correct the STS api call there.

**Comment by monde** [comment]
https://github.com/okta/okta-aws-cli/pull/174 addresses this issue. I'll close this issue when that is released this weak on the v2.1.0 release.

**Comment by monde** [comment]
Addressed in https://github.com/okta/okta-aws-cli/releases/tag/v2.1.0 with the `--aws-region` CLI flag
